lib/plottr_components/src/components/characters/CharacterListView.js

```diff
@@ -305,6 +305,23 @@ const CharacterListViewConnector = (connector) => {
     const renderVisibleCharacters = (categoryId) => {
       if (!visibleCharactersByCategory[categoryId]) return []
 
+      if (!!attributeTabId && attributeTabId !== 'all') {
```

Comment:
This needs to be a selector.
If the selector that we're using doesn't have a name that implies that it has this "filtering by book" functionality, then please add another selector.

Reply:
@Quiescent got it thank you! 🙇‍♀️